speechbrain/processing/features.py

```diff
@@ -1316,11 +1515,45 @@ def _load(self, path, end_of_epoch=False):
             Here for compatibility, but not used.
         """
         del end_of_epoch  # Unused here.
-        device = "cpu"
-        stats = torch.load(path, map_location=device)
+        stats = torch.load(path, map_location=self.device)
         self._load_statistics_dict(stats)
 
 
+def get_mask(x, lengths=None, length_dim=1):
```

Comment:
We have 1 billion function doing this in SB, I wonder if we should do smth about it.